### PR TITLE
removed requirement of lib2to3

### PR DESCRIPTION
**Pull request by goulu**
gml.py doesn't use lib2to3 anymore, 
so lib2to3 isn't required anymore in networkx
so programs using networkx can be "compiled" using py2exe

see https://github.com/networkx/networkx/issues/1775

### CODE REVIEW

File: networkx/readwrite/gml.py [hunk 1]
```diff
@@ -134,14 +130,12 @@ def literal_destringizer(rep):
         If ``rep`` is not a Python literal.
     """
     if isinstance(rep, (str, unicode)):
-        orig_rep = rep
+        orig_rep=rep
+        if rtp_fix_unicode:
+            rep=rtp_fix_unicode(rep)
```
Comment (jfinkels) [inline networkx/readwrite/gml.py]: Please put spaces around the assignment operator.

File: networkx/readwrite/gml.py [hunk 2]
```diff
@@ -134,14 +130,12 @@ def literal_destringizer(rep):
         If ``rep`` is not a Python literal.
     """
     if isinstance(rep, (str, unicode)):
-        orig_rep = rep
+        orig_rep=rep
+        if rtp_fix_unicode:
```
Comment (jfinkels) [inline networkx/readwrite/gml.py]: Please use `if rtp_fix_unicode is None:`.

File: networkx/readwrite/gml.py [hunk 3]
```diff
@@ -134,14 +130,12 @@ def literal_destringizer(rep):
...
+            rep=rtp_fix_unicode(rep)
         try:
-            # Python 3.2 does not recognize 'u' prefixes before string literals
-            if rtp_fix_unicode:
-                rep = str(rtp_fix_unicode.refactor_string(
-                    rep + '\n', '<string>'))[:-1]
             return literal_eval(rep)
-        except (ParseError, SyntaxError, TokenError):
+        except (SyntaxError):
```
Comment (jfinkels) [inline networkx/readwrite/gml.py]: Parentheses are unnecessary here.

**Comment by hagberg** [comment]
@goulu would you make these small fixes so we can merge?

**Pull request merged**